app/controllers/admin/dashboard_controller.rb

```diff
@@ -51,7 +51,6 @@ def initialize_wice_grid(projects, person_id, page)
     initialize_grid(
         projects,
         include: [:primary_agent, :secondary_agent],
-        per_page: page,
```

Comment:
I'm not sure if the client will want to specify a limit in the future, But we can always add this back in in the future, if needed.